flink-filesystems/flink-s3-fs-presto/src/main/resources/META-INF/NOTICE

```diff
@@ -57,3 +57,8 @@ This project bundles the following dependencies under BSD License (https://opens
 See bundled license files for details.
 
 - org.codehaus.woodstox:stax2-api:3.1.4 (https://github.com/FasterXML/stax2-api/tree/stax2-api-3.1.4)
+
+This project bundles the following dependencies under the CDDL 1.1 license.
+See bundled license files for details.
+
+- javax.xml.bind:jaxb-api:2.3.1
```

Comment:
given that this depends on javax.activation I would think that we need to bundle both.
We had an issue in the past when we only had one in yarn: https://issues.apache.org/jira/browse/FLINK-15008